lxd/storage_pools_config.go

```diff
@@ -128,30 +132,14 @@ func storagePoolFillDefault(name string, driver string, config map[string]string
 		config["size"] = strconv.FormatUint(uint64(size), 10)
 	}
 
-	if driver == "lvm" {
-		if config["lvm.thinpool_name"] == "" {
-			config["lvm.thinpool_name"] = "LXDThinpool"
-		}
-
-		if config["volume.block.filesystem"] == "" {
-			config["volume.block.filesystem"] = "ext4"
-		}
-
-		if config["volume.block.mount_options"] == "" {
-			config["volume.block.mount_options"] = "discard"
-		}
-	}
-
-	if config["volume.size"] == "" {
-		if driver == "lvm" {
-			sz, err := shared.ParseByteSizeString("10GB")
+	if driver == "lvm" || driver == "btrfs" || driver == "zfs" {
+		if config["volume.size"] != "" {
```

Comment:
So quotas are a bit annoying:
 - For images, they don't make sense, don't apply any.
 - For custom volumes and containers, it's a bit tricky as the preferred way to set such quotas would be through the device disk entry (size property) so that the size can be controlled through profiles.